### PR TITLE
Clarify some documentation.

### PR DESCRIPTION
**Pull request by jbeda**
Also fix up softlinks that got lost during repo reorganization.  Fixed a
typo in example workload.

Signed-off-by: Joe Beda <joe.github@bedafamily.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ What you do need:
 - A Kubernetes cluster that supports Service objects of `type: LoadBalancer` ([AWS Quickstart cluster][9] or Minikube, for example)
 - `kubectl` configured with admin access to your cluster
 
-See the [deployment documentation][10] for more deployment options.
+See the [deployment documentation][10] for more deployment options if you don't meet these requirements.
```
Comment (Bradamant3) [inline README.md]: A nit, and a hill I won't die on, but the added phrase isn't necessary. Its meaning is inherent in the original. If you think it adds clarity, keep it.

File: README.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ How you configure DNS depends on your platform:
 
 ### More information and documentation
 
-For more deployment options, see the [deployment documentation][10].
+For more deployment options, including uninstalling Contour, see the [deployment documentation][10].
```
Comment (Bradamant3) [inline README.md]: I should have caught this before -- we should probably move the uninstall instructions into the readme. Makes more sense -- it's not a deployment option ;). I can do it when I rework Ralph's content if you like.

File: docs/deploy-options.md [hunk 3]
```diff
@@ -2,12 +2,19 @@
 
 The [README][0] shows you a simple way to get started with Contour on your cluster. This topic explains the details and shows you additional options.
 
+## Manual and Specialized Deployments
+
+This document outlines how to run Contour using a Kubernetes Service of `Type: LoadBalancer`.  If you don't have a cluster with that capability or if you don't want to use it you have two other options:
+
+* Use a Kubernetes Service with `Type: NodePort`.  This configure every node in your cluster to forward traffic on that Node to Contour.
```
Comment (Bradamant3) [inline docs/deploy-options.md]: This one is also documented below (##Running without a Kubernetes LoadBalancer). Do we need it in both places? The more detailed version is more helpful, too.
But if you prefer this (shorter) version, it still needs minor edits:
"This Service configures every node in your cluster to forward traffic on the node to Contour." (the generic reference to "node")
OR
"This Service configures every Node in your cluster to forward traffic on the Node to Contour." (the specific object reference. consistency ...)

File: docs/deploy-options.md [hunk 4]
```diff
@@ -2,12 +2,19 @@
...
+* Don't use a Kubernetes Service at all.  You can run with `hostNetwork: true` on your pod and direct traffic to port 8080 on the nodes that are running Contour. See the [AWS NLB tutorial][3] as an example.
```
Comment (Bradamant3) [inline docs/deploy-options.md]: And then if we keep the current content in its current location, we should move this one down to the section ##Running without ...

File: README.md (hunk 1 above)
Comment (jbeda) [reply README.md]: Someone on twitter asked about this specifically and so I wanted to add more color.

File: README.md (hunk 2 above)
Comment (jbeda) [reply README.md]: We probably want it both places?  In any case, I'll let you fix it as you rework Ralph's stuff.

File: docs/deploy-options.md (hunk 3 above)
Comment (jbeda) [reply docs/deploy-options.md]: Merging this into section below.

File: docs/deploy-options.md (hunk 4 above)
Comment (jbeda) [reply docs/deploy-options.md]: yup

**Comment by jbeda** [comment]
@Bradamant3 PTAL when you get a chance.

**Comment by Bradamant3** [comment]
Nice changes/fixes :-)

File: README.md (hunk 1 above)
Comment (Bradamant3) [reply README.md]: The best kind of reason :-)

**Pull request merged**